node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -170,7 +170,11 @@ object Configuration {
       case Some(options.diagnostics) => Diagnostics
       case Some(options.deploy) =>
         import options.deploy._
-        Deploy(from(), phloLimit(), phloPrice(), nonce(), location())
+        Deploy(from.getOrElse("0x"),
+               phloLimit.getOrElse(0),
+               phloPrice.getOrElse(0),
+               nonce.getOrElse(0),
+               location())
```

Comment:
Can we add a TODO to change these defaults? We definitely don't want the default phloPrice/phloLimit to be 0 for mainnet but we still haven't figured out what the appropriate limits are for a regular deploy.

Reply:
Added.